### ISSUE TITLE
Convert ANTs commands to use antspy

### ISSUE THREAD

**Issue by mikarubi** [mikarubi/voluseg#32]
since we got rid of spark as a dependency, it makes sense to do same for ants, since it has a simple wrapper in python. https://pypi.org/project/antspyx/

it should be straightforward to convert the corresponding ants calls into python. i can give it a shot, and the advantage would be a package that can be fully installed via pip without need to install other software.

_Originally posted by @mikarubi in https://github.com/mikarubi/voluseg/issues/26#issuecomment-2326873652_

**Comment by luiztauffer** [comment]
that would be great! Users wouldn't need to manually install and set PATH to ANTs

**Comment by luiztauffer** [comment]
As of now, the tests are mostly checking if the code runs successfully and asserting that the expected files are created. I suggest we start including assertions on the content of these files, to guarantee that such a modification will work as expected

**Comment by mikarubi** [comment]
Agree. I believe that the pipeline is largely deterministic, except for:

step 2: ants_registration (https://github.com/ANTsX/ANTs/wiki/antsRegistration-reproducibility-issues)).
step 3: GaussianMixture
step 4: AgglomerativeClustering
step 4: linalg.lstsq

For the purposes of such a test, we could run omit set `registration='none'` and set random number seeds to ensure reproducibility in other steps.
